root_VS2010/programs/VB/z_Common2.bat

```diff
@@ -5,18 +5,18 @@
 @rem --------------------------------------------------
 @rem Set Program Files path
 @rem --------------------------------------------------
-reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set PROGRAM_FILES=Program Files || set PROGRAM_FILES=Program Files (x86)
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set PROGRAM_FILES=C:\Program Files || set PROGRAM_FILES=C:\Program Files (x86)
```

Comment:
@daisukenishino 

We have created pull request to nishino branch. Please merge
https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/63
https://github.com/SymphonyTeleca/OpenTouryo/pull/87